Validation/MtdValidation/scripts/ROC_plotter.py

```diff
@@ -0,0 +1,319 @@
+import numpy as np
+import uproot as up
+import matplotlib.pyplot as plt
+from matplotlib.ticker import NullFormatter
+
+"""
+This is a sample code for ROC curve plotting (Feel free to change it to Your needs!)
+Code is ran from "Validation/MtdValidation/test" directory
+Before running the code initialize cmsenv
+Run the code by using python3 -> 'python3 ROC_ploter.py'
+Change the DQM File and ROC plot output directories accordingly to Your workspace
+
+"""
+
+
+print("All libraries has been read in!")
+
+directory_path = '/afs/cern.ch/user/n/nstrautn/CMSSW_13_1_0_pre4/src/Validation/MtdValidation/test/' # DQM file location directory
+ROC_plots_directory = '/afs/cern.ch/user/n/nstrautn/CMSSW_13_1_0_pre4/src/Validation/MtdValidation/test/ROC/' # ROC plots location directory
```

Comment:
Can we do something like this?
 `$CMSSW_BASE/src/Validation/MtdValidation/test/ROC/` ?

Reply:
maybe with `os`:
```
import os
base = os.getenv('CMSSW_BASE')
directory_path = base + '/src/Validation/MtdValidation/test/'
```

Reply:
Dear all, this is just a simple example, to be anyway edited depending on what is really needed, also for what concerns plots. Anyway we may update it as suggested